modules/uhttp/client/client_middlewares_benchmark_test.go

```diff
@@ -34,26 +34,26 @@ import (
 	jconfig "github.com/uber/jaeger-client-go/config"
 )
 
-// BenchmarkClientFilters/empty-8         	  500000	      3517 ns/op	     256 B/op	       2 allocs/op
-// BenchmarkClientFilters/tracing-8       	   20000	     64421 ns/op	    1729 B/op	      29 allocs/op
-// BenchmarkClientFilters/auth-8          	   50000	     36574 ns/op	     728 B/op	      16 allocs/op
-// BenchmarkClientFilters/default-8       	   10000	    104374 ns/op	    2275 B/op	      43 allocs/op
-func BenchmarkClientFilters(b *testing.B) {
+// BenchmarkClientMiddlewares/empty-8          100000000           10.8 ns/op         0 B/op          0 allocs/op
```

Comment:
#239 reduced allocations and I forgot to update here. The same benchmark result is written is 3 places like: README.MD, doc.go and client_middlewares_benchmark_test.go